README.md

```diff
@@ -17,6 +17,8 @@ To run the repository you will need to configure a Snowflake connection. With at
 - password: your Snowflake password
 - location: your Snowflake location (e.g. us-east-01)
```

Comment:
Isn't this field called "region"?

Reply:
Huh, yes it is. I have no idea how I came up with `location` here 😅